models/load_conf.py

```diff
@@ -45,6 +46,14 @@ def _to_markdown(html: str, **options: Any) -> str:
     return cast(str, ConferenceMarkdownConverter(**options).convert(html))
 
 
+def clean_text(markdown_content: str) -> Any:
```

Comment:
We actually want to keep this text. I use it later to get links to specific paragraphs in the page. So in the UI, if someone clicks on a link to a conference talk, they are taken directly to that paragraph in the talk.